src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -193,6 +193,12 @@ public void updateFilteredMeetingList(Predicate<Meeting> predicate) {
         return filteredPersons.sorted(Person::compareTo);
     }
 
+    /*
+    @Override
+    public ObservableList<Meeting> getFilteredMeetingList() {
```

Comment:
Also not really sure why this is a comment

Reply:
I think this needs to be there?

Reply:
There were some discrepancies when I merged the team repo master branch so I think thats why there are some random comments here and there. I have removed them alr.